contracts/Core/parameters/child/StakeManagerParams.sol

```diff
@@ -23,6 +23,7 @@ abstract contract StakeManagerParams is ACL, IStakeManagerParams, Constants {
     SlashNums public slashNums = SlashNums(500, 9500, 0);
     // Slash Penalty = bounty + burned + kept
     uint256 public minStake = 1000 * (10**18);
+    uint256 public minSafeRazor = 1 * (10**18);
```

Comment:
hey 
all looks good 
Just increase this value from 1 Razor to Maybe something significant (100 maybe)
hmm maybe this is okay as well

(2 ^ 32 -1) * 0.4 (CurrentRazorValue) = 1,71,79,86,918

cool approving, thanks

Reply:
current razor value is 0.04 not 0.4
lets make it 10000e18 Razor ($400)